quickevent/app/plugins/Competitors/src/competitorwidget.cpp

```diff
@@ -347,8 +373,7 @@ void CompetitorWidget::loadFromRegistrations(int siid)
 		onRegistrationSelected(vals);
 	}
 	else {
-		qf::core::model::DataDocument*doc = dataController()->document();
-		doc->setValue(QStringLiteral("competitors.siid"), siid);
+		dataController()->document()->setValue(QStringLiteral("competitors.siid"), siid);
```

Comment:
why this change?

Reply:
simplification

Reply:
never do changes not related to implemented functionality, think about the reviewer, keep his life beautiful